scp/cc/cpio/client_providers/blob_storage_client_provider/src/BUILD.bazel

```diff
@@ -23,10 +24,13 @@ cc_library(
             "//:aws_platform": [
                 "//scp/cc/cpio/client_providers/blob_storage_client_provider/src/aws:aws_blob_storage_client_provider_lib",
             ],
+            "//:azure_platform": [
+                "//scp/cc/cpio/client_providers/blob_storage_client_provider/src/gcp:gcp_blob_storage_client_provider_lib",
+            ],
```

Comment:
**[Comment for actual PR]**
We defined `azure_platform` for `blob_storage_client_provider_select_lib` using the GCP implementation under `/src/gcp`. We think it's not used in B&A services at this moment, but we added it just to avoid build errors when we run `build_and_test_all_in_docker`. We will find a better solution for that problem in a future PR.